pkg/chartutil/requirements.go

```diff
@@ -106,3 +116,144 @@ func LoadRequirementsLock(c *chart.Chart) (*RequirementsLock, error) {
 	r := &RequirementsLock{}
 	return r, yaml.Unmarshal(data, r)
 }
+
+// ProcessRequirementsConditions disables charts based on condition path value in values
+func ProcessRequirementsConditions(reqs *Requirements, cvals Values) {
+	var cond string
+	var conds []string
+
+	if reqs != nil && len(reqs.Dependencies) > 0 {
+		for _, r := range reqs.Dependencies {
+			var hasTrue, hasFalse bool
+			cond = string(r.Condition)
+			// check for list
+			if len(cond) > 0 {
+				if strings.Contains(cond, ",") {
+					conds = strings.Split(strings.TrimSpace(cond), ",")
+				} else {
+					conds = []string{strings.TrimSpace(cond)}
+				}
+				for _, c := range conds {
+					if len(c) > 0 {
+						// retrieve value
+						vv, err := cvals.PathValue(c)
+						if err == nil {
+							if vv.(bool) {
```

Comment:
Good point. If a user types `"true"` (with quotes) that would cause a panic. We've seen that happen before. So maybe: 
```
bval, ok := vv.(bool)
if !ok {
  // return an error about only bools supported
}
if bval {
    // ... and so on...
}
```